src/lib/webpack-config.js

```diff
@@ -105,8 +105,8 @@ export default env => {
 					pkg = readJson(manifest) || {};
 				return !!(pkg.module || pkg['jsnext:main']);
 			},
-			babelrc: false,
-			...createBabelConfig(env)
+			babelrc: true,
```

Comment:
Yeah, nah - I've checked this it works for the cases specified in #9, the ```exlude: []``` messes things up - babelrc is innocent 😛.